deploy/docs/Installation_with_Helm.md

```diff
@@ -81,6 +81,25 @@ For Helm3, if the namespace does not exist, you can add the `--create-namespace`
 helm upgrade --install my-release sumologic/sumologic --namespace=my-namespace --set sumologic.accessId=<SUMO_ACCESS_ID> --set sumologic.accessKey=<SUMO_ACCESS_KEY>  --set sumologic.clusterName="<MY_CLUSTER_NAME> --create-namespace"
 ```
 
+### Installing the helm chart in Openshift platform
+The daemonset/statefulset fails to create the pods in Openshift environment due to the request of elevated privileges, like HostPath mounts, privileged: true, etc.
+
+If you wish to install the chart in the Openshift Platform, it requires a SCC resource which is only created in Openshift (detected via API capabilities in the chart), you can do the following:
+
+1. Set `sumologic.scc.create` to `true`.
+
+2. Uncomment the below `securityContext` in the fluent-bit config section:
+```
+## Set securityContext of fluent-bit daemonset pods as privileged for running in Openshift
```

Comment:
For consistency, can we not, in the section document --set for both flags? That way, users can refer to the customize installation step for the values.YAML update? Also, this should be updated in each of the different Prometheus variants.

Reply:
done